tool/flutter_pub_upgrade.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+# Backs up one directory at a time, looking for one called "flutter". Once it
+# finds that directory, an absolute path to it is returned.
+function getFlutterPath() {
```

Comment:
When I was thinking about this script before, I imagined it as something we'd run locally to update all the apps at once, rather than something that Travis would run.

If that's your idea as well, all this localSdkPath code can be taken out, since `flutter` should just be in $PATH.

Reply:
Done